src/EditDialog.cpp

```diff
@@ -21,6 +21,591 @@
 #include <QPainter>
 #include <QClipboard>
 
+//  The following section of code should really be in a separate class
```

Comment:
Why two spaces instead of just one?

Reply:
Just my style. Goes back to the days of internal documentation and the 'docment' program on CDC mainframes.